From 9542715a05c2d7e5205b3d458e7b3987da330665 Mon Sep 17 00:00:00 2001
From: Akihiro Motoki <amotoki@gmail.com>
Date: Sat, 11 Nov 2017 22:47:49 +0000
Subject: [PATCH 16/29] Move django variants UT and selenium-headless to
 in-repo jobs

Note that horizon-openstack-tox-py27dj18 is unnecessary for Ocata
because django 1.8 is the default Django version in Ocata.

Change-Id: Ic040bb1bf3e6c177dd88f2304491b7e839555e76
(cherry picked from commit ee3e6127a14032c7d2b8f23971cdaedb8cc51fe0)
---
 .zuul.yaml | 40 ++++++++++++++++++++++++++++++++++++++++
 1 file changed, 40 insertions(+)
 create mode 100644 .zuul.yaml

diff --git a/.zuul.yaml b/.zuul.yaml
new file mode 100644
index 0000000..3b695be
--- /dev/null
+++ b/.zuul.yaml
@@ -0,0 +1,40 @@
+- job:
+    name: horizon-openstack-tox-base
+    parent: openstack-tox
+    irrelevant-files:
+      - ^.*\.rst$
+      - ^doc/.*$
+      - ^releasenotes/.*$
+      - ^horizon/locale/.*$
+      - ^openstack_dashboard/locale/.*$
+
+- job:
+    name: horizon-openstack-tox-py27dj19
+    parent: horizon-openstack-tox-base
+    vars:
+      tox_envlist: py27dj19
+
+- job:
+    name: horizon-openstack-tox-py27dj110
+    parent: horizon-openstack-tox-base
+    vars:
+      tox_envlist: py27dj110
+
+- job:
+    name: horizon-selenium-headless
+    parent: horizon-openstack-tox-base
+    vars:
+      tox_envlist: selenium-headless
+
+- project:
+    name: openstack/horizon
+    check:
+      jobs:
+        - horizon-openstack-tox-py27dj110
+        - horizon-openstack-tox-py27dj19
+        - horizon-selenium-headless
+    gate:
+      jobs:
+        - horizon-openstack-tox-py27dj110
+        - horizon-openstack-tox-py27dj19
+        - horizon-selenium-headless
-- 
1.8.3.1

